### ISSUE TITLE
Python: Digifinex subaccount fetch deposits

### ISSUE THREAD

**Issue by Peter416** [ccxt/ccxt#14507]
I am using Python to get my deposits through Digifinex using a sub account. I am trying the following 

`exchange_id = 'digifinex'
exchange_class = getattr(ccxt, exchange_id)
exchange = exchange_class({
    'apiKey': "API_KEY",
    'secret': "SECRET_KEY"
})

exchange.fetchDeposits()`

But I get the following error: `RequestTimeout: digifinex GET https://openapi.digifinex.com/v3/currencies` 

How should I fetch the deposits for the sub account? I could not find any documentation on that

**Comment by carlosmiei** [comment]
Hi @Peter416,
Can you please try again? Usually, a `RequestTimeout` error means that you had some temporary network issues.

**Comment by Peter416** [comment]
Hi @carlosmiei ,
I tried to run several times and got the same error. Do you know of any documentation to deal with a sub account?

**Comment by carlosmiei** [comment]
@Peter416 is the error the same? It's very weird if you're getting the same, it's not related to the deposits endpoint but an initial action CCXT performs to load the markets. 

You can also check Digifinex API docs here: https://docs.digifinex.com/

**Comment by Peter416** [comment]
@carlosmiei yes it's the same error. I couldn't find anything that would help in the documentation. Does ccxd have any documentation for accessing sub accounts?

**Comment by carlosmiei** [comment]
@Peter416  we don't but that error is not related to accounts/sub-accounts, that endpoint is even public. Can you please try to open this link in the browser to see if it's working there? https://openapi.digifinex.com/v3/currencies

**Comment by Peter416** [comment]
@carlosmiei yes the link opens. I still can't seem to figure out what the issue is. Do you know if an extra parameter needs to be added to my code above? i.e. a `headers` parameter?
